### DISCUSSION TITLE
windows 10 error json error

### DISCUSSION THREAD

**Discussion by zhekafun** [Chia-Network/chia-blockchain#3182] [General]
Version: 1.1.2 
win 10
in developers tools console got error
`DevTools failed to load SourceMap: Could not parse content for file:///C:/Users/pc/AppData/Local/chia-blockchain/app-1.1.2/resources/app.asar/build/renderer/js/main.js.map: Unexpected end of JSON input`

**Comment by jgabriel98** [comment]
same

**Comment by aqk** [comment]
Is this when you are starting the app?

Please install https://keybase.io/download
And then come to keybase://chat/chia_network.public
Join #support for support or #dev for more discussion on development.

Could you give a little more info about what you are trying to accomplish?
